sql/item_mods.sql

```diff
@@ -2043,8 +2819,22 @@ INSERT INTO `item_mods` VALUES (10955,135,2);
 INSERT INTO `item_mods` VALUES (10957,28,8);
 INSERT INTO `item_mods` VALUES (10957,30,2);
 INSERT INTO `item_mods` VALUES (10958,165,3);
+INSERT INTO `item_mods` VALUES (10959,90,8); -- INQUISITOR'S CHAIN, Club skill +8 Divine magic skill +5 Healing magic skill +5
+INSERT INTO `item_mods` VALUES (10959,111,5); -- DIVINE SKILL
+INSERT INTO `item_mods` VALUES (10959,112,5); -- HEALING SKILL
 INSERT INTO `item_mods` VALUES (10960,374,4);
 INSERT INTO `item_mods` VALUES (10960,375,4);
+INSERT INTO `item_mods` VALUES (10961,2,50); -- LAVALIER, HP+50 Charmed:[Element:Light]-40 [Element:Dark]-40
+-- INSERT INTO `item_mods` VALUES (10961,CHARMED:LIGHT RESIST,-40);
```

Comment:
This can go in the item_latents table. Check latent_effect.h for the condition you want (ie. LATENT_STATUS_EFFECT_ACTIVE     = 13, //status effect on player - PARAM: EFFECTID)